### ISSUE TITLE
`namespace` not working

### ISSUE THREAD

**Issue by VinSpee** [davidtheclark/stylelint-selector-bem-pattern#6]
Hi! I'm migrating to this from postcss-bem-linter and running into an issue with namespacing. My config looks like this:

```json
{
  "extends": "stylelint-config-suitcss",
  "plugins": [
    "stylelint-selector-bem-pattern"
  ],
  "rules": {
    "selector-bem-pattern": {
      "preset": "suit",
      "presetOptions": {
        "namespace": "qz"
      }
    },
    "indentation": "tab",
    "number-leading-zero": null,
    "no-missing-eof-newline": null
  }
}
```

when I run the command, I get 

```
Invalid value "[object Object]" for option "presetOptions" of rule "selector-bem-pattern"
```

are my presetOptions properly formatted? Any idea what could be amiss?

**Comment by VinSpee** [comment]
thank you sir! amazing work as always.

**Comment by davidtheclark** [comment]
Thanks for finding and reporting the bug. I guess the stylelint options validator function was not ready for deeply nested objects.

**Comment by davidtheclark** [comment]
Released in 0.2.1
